src/osal/rt-kernel/osal.c

```diff
@@ -100,6 +100,8 @@ os_thread_t * os_thread_create (const char * name, int priority,
    return task_spawn (name, entry, priority, stacksize, arg);
 }
 
+/********************** Mutex ************************************************/
```

Comment:
Please keep this file similar to other osal.c implementations (in this project and others), to simplify maintenance.

Reply:
Done